Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -39,7 +40,14 @@
 ActsExamples::TrackFindingAlgorithm::TrackFindingAlgorithm(
     Config config, Acts::Logging::Level level)
     : ActsExamples::IAlgorithm("TrackFindingAlgorithm", level),
-      m_cfg(std::move(config)) {
+      m_cfg(std::move(config)),
+      m_trackSelector([this]() -> std::optional<Acts::TrackSelector> {
+        if (m_cfg.trackSelectorCfg.has_value()) {
+          return {m_cfg.trackSelectorCfg.value()};
+        } else {
+          return std::nullopt;
+        }
+      }()) {
```

Comment:
this looks a bit tedious but I dont see a better solution

Reply:
🤷